CodeMaid/CodeMaid.vsct

```diff
@@ -288,13 +288,15 @@
           If you do not want an image next to your command, remove the Icon node or set it to <Icon guid="guidOfficeIcon" id="msotcidNoIcon" /> -->
       <Button guid="GuidCodeMaidCommandAbout" id="CmdIDCodeMaidAbout" priority="0x0100" type="Button">
         <Icon guid="GuidImageInfo" id="IconInfo" />
+        <CommandFlag>TextChanges</CommandFlag>
```

Comment:
Except for the dynamic text, I didn't find a better way to do it.There seems to be an article that says VSCT can have multiple languages. But I don't know how to do it so far.

Reply:
I think looking into this further will be necessary.. in the past the performance hit was debilitating as our extension as well as other ones would cause each other to infinitely regenerate the text.

Reply:
I tried the method described in MSDN, but I didn't succeed and I didn't know what the problem was. 
url: https://msdn.microsoft.com/en-us/library/ee943168.aspx

Reply:
3642dec9c02dbf92e4748993daa093df86fa7d0b